packages/core/getBusyTimes.ts

```diff
@@ -27,7 +39,6 @@ export async function getBusyTimes(params: {
     .findMany({
       where: {
         userId,
-        eventTypeId,
```

Comment:
Nice spot @CarinaWolli  - Looks like we were only getting busy events for this particular event type and not all event types.  Potentially allowing a double booking (same time) as each other